lib/active_model/serializer.rb

```diff
@@ -143,29 +166,10 @@ def attributes(options = {})
       end
     end
 
-    def self.serializers_cache
-      @serializers_cache ||= ThreadSafe::Cache.new
-    end
-
-    def self.digest_caller_file(caller_line)
-      serializer_file_path = caller_line[CALLER_FILE]
-      serializer_file_contents = IO.read(serializer_file_path)
-      Digest::MD5.hexdigest(serializer_file_contents)
-    end
-
-    attr_reader :options
+    private # rubocop:disable Lint/UselessAccessModifier
```

Comment:
I think this is how class method are supposed to be declared private: http://stackoverflow.com/a/4953043/356849